django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -2122,8 +2219,16 @@ def test_order_by_genre(self):
 
     def test_order_by_cantus_id(self):
         source = make_fake_source()
-        chant_1 = make_fake_chant(incipit="isaac", cantus_id="121393", source=source)
-        chant_2 = make_fake_chant(incipit="baal", cantus_id="196418", source=source)
+        chant_1 = make_fake_chant(
+            manuscript_full_text_std_spelling="isaac", cantus_id="121393", source=source
+        )
+        chant_2 = make_fake_chant(
+            manuscript_full_text_std_spelling="baal", cantus_id="196418", source=source
+        )
+        chant_1.refresh_from_db()
```

Comment:
Not sure this is worth it at this point, but do you always need to `.refresh_from_db()` once you've created the chant. Why not return the refreshed version from `make_fake_chant`? Then you don't need this and the accompanying comment everywhere...

Reply:
This is a good observation/suggestion. I've implemented this!